### PR TITLE
changed README

### PR DESCRIPTION
**Pull request by frankystone**
This is just a suggestion :-)

But i have some more questions, problems to understand some things. Will ask those questions later.

Can you generally agree to this changes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,41 +3,59 @@ Repository faking an add-ons server for [@widelands](https://github.com/wideland
...
+Same applies to screenshots: Just replace or add new ones. Be sure to update the `descriptions` file.
+
+After you have done run `java UpdateList`, then git add,commit,push.
 
 ### Verifying add-ons
 
-Run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
+Updating an add-on will automatically set it to Not Verified. It is not allowed to mark an add-on as verified during the update.
+To verify it run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
+
```
Comment (frankystone) [inline README.md:41]: I verifying done automatically?

File: README.md [hunk 2]
```diff
@@ -3,41 +3,59 @@ Repository faking an add-ons server for [@widelands](https://github.com/wideland
...
+### Uploader
 
-### Notes
+The name of the uploader of a new add-on is automatically set to `"Nordfriese"` – if you are not me, consider editing the list manually after running `buildcats.sh` or `java UpdateList` to replace it with your name. (Each add-on's uploader is listed directly below the author's name.)
```
Comment (frankystone) [inline README.md:45]: As far i understand running buildcats.sh isn't needed to get, yes what?

Where do i have to modify the uploader? I guess in the file called list, or list_2, or both?

Comment (Noordfrees) [reply README.md:45]: `buildcats.sh` is used for the translations updates, to sync the PO source files with the downloadable MO files. This script is now run automatically by a GitHub action every night, so it needs to be run manually only when I am doing some maintenance that messes up the repo structure.

`list_2` is the version parsed by the current Widelands master; `list` has a slightly different format used by "older" Widelands versions. The uploader name should be edited in both files.

File: README.md (hunk 1 above)
Comment (Noordfrees) [reply README.md:41]: Verifying means that a developer looks at the add-on in question and decides that it is safe for use; then he runs this script and makes a commit with the change. The add-on will then get the Verified badge in the add-ons manager.

**Comment by frankystone** [comment]
Thanks for answering my questions :-)

I think this is ready for review now. [How it will look like](https://github.com/frankystone/wl_addons_server/tree/README_for_dummies)

File: README.md [hunk 3]
```diff
@@ -3,53 +3,72 @@ Repository faking an add-ons server for [@widelands](https://github.com/wideland
 
 I don’t know how long this "server" will be in use, so I’d better write some brief documentation about maintenance here.
 
-### Adding a new add-on
+If you want to contribute a new add-on, [informations how to create add-ons can be found here.](https://github.com/widelands/widelands/blob/master/doc/sphinx/source/add-ons.rst)
```
Comment (Noordfrees) [inline README.md:6]:
```suggestion
If you want to contribute a new add-on, [information how to create add-ons can be found here.](https://github.com/widelands/widelands/blob/master/doc/sphinx/source/add-ons.rst)
```

File: README.md [hunk 4]
```diff
@@ -3,53 +3,72 @@ Repository faking an add-ons server for [@widelands](https://github.com/wideland
...
 Keep in mind that currently some of the add-ons here are meant as proof-of-concept dummies. Please do not report any errors regarding those.
 
 Issues not directly related to add-ons should be reported [against the official game](https://github.com/widelands/widelands/issues).
 
-### Contributing
+### Developers corner
+
+#### Verify a new add-on
+
+To verify a new add-on read the code carfully and make sure it does not contain malicious code. Then run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
```
Comment (Noordfrees) [inline README.md:57]:
```suggestion
To verify a new add-on, read the code carefully and make sure it does not contain malicious code. Then run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
```

File: README.md [hunk 5]
```diff
@@ -3,53 +3,72 @@ Repository faking an add-ons server for [@widelands](https://github.com/wideland
...
+When you updated any of the files in `po/` or added new ones, just do git add,commit,push.
+
+As mentioned above `buildcats.sh` shouldn't be run manually. If you have to do it anyway for some reason, do this **only** on the master branch.
+
+#### Notes
+
+`buildcats.sh` usually prints some errors which are false alarms. This script is not clever enough to catch translatable strings in config scripts (such as a map’s `elemental` file), except for the `addon` file.
 
-If you want to contribute a new add-on, update an existing one, or submit new or updated translations, open a pull request with this change and make sure you followed the steps described above.
+If you want to update an existing one, or submit new or updated translations, open a pull request with this change and make sure you followed the steps described above.
```
Comment (Noordfrees) [inline README.md:71]: It is not clear what "an existing one" refers to here, IMHO

**Review by Noordfrees — COMMENTED** [review]
LGTM, three comments inline

**Comment by frankystone** [comment]
Thanks :-)

All done.

**Review by Noordfrees — APPROVED** [review]

**Comment by frankystone** [comment]
Thanks!

**Pull request merged**